src/compensations/Compensations.tsx

```diff
@@ -121,19 +128,27 @@ const Compensations = ({ compensations, locations }: CompensationsProps) => {
           />
           {salary !== null ? (
             <div aria-live="polite">
-              <Text> Du vil få en årlig lønn på {salary}</Text>
+              <Text>
+                {" "}
+                Du vil få en årlig lønn på{" "}
+                {formatAsCurrency(salary, locale.locale, locale.currency)}
```

Comment:
just a preference, but I fint it easier to read:
```suggestion
                ´ Du vil få en årlig lønn på ${{formatAsCurrency(salary, locale.locale, locale.currency)}}´
```